gemp-lotr/gemp-lotr-async/src/main/web/js/gemp-022/jCards.js

```diff
@@ -29,7 +29,7 @@ var packBlueprints = {
     "(S)SH - Starter": "/gemp-lotr/images/boosters/sh_starter_selection.png",
```

Comment:
only allow styling changes and comments into this file and any pre-existing files. They should remain holy sources of truth and stability for players :)